app/tests/unit/pages/project-revision/[projectRevision]/index.test.tsx

```diff
@@ -157,10 +157,27 @@ describe("The Create Project page", () => {
 
   it("Renders the summary with the filled-out details", async () => {
     pageTestingHelper.loadQuery({
-      Form() {
-        return {
-          jsonSchema: projectContactProdSchema,
-        };
+      Form(context) {
+        // If the form is the project manager form, return the project manager schema
+        if (
+          context.name === "formByJsonSchemaName" &&
+          context.path[2] == "projectManagerFormChangesByLabel"
+        ) {
+          return {
+            jsonSchema: projectManagerProdSchema,
+          };
+        }
+        // If the form is the project contact form, return the project contact schema
+        if (
+          context.name === "formByJsonSchemaName" &&
+          context.path[2] == "summaryContactFormChanges"
+        ) {
+          return {
+            jsonSchema: projectContactProdSchema,
+          };
+        }
+        // Handle default case
+        return { jsonSchema: projectContactProdSchema };
```

Comment:
Might be better to return `null` or `undefined` if the form isn't given?

Reply:
Using null/undefined will cause the test to break.  Once we have context for all use cases we can adjust to a switch statement to eliminate this line.

Reply:
tried that too - no luck.  My suggestion is to create a tech debt to refactor the if/else to switch case once we've completed all the other forms.

Reply:
Added an entire schema object - should do the trick for now.